### DISCUSSION TITLE
OMI glTF Working Group Meeting 2022-08-04

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#98] [General]
This meeting will be on 8/04/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:

- [gltf extensions](https://github.com/omigroup/gltf-extensions) Discuss support table
- glTF Transform Updates
- [KHR_audio](https://github.com/KhronosGroup/glTF/pull/2137) Feedback
- [OMI_collider](https://github.com/omigroup/gltf-extensions/pull/63)
- OMI_id / [OMI_ref](https://github.com/omigroup/gltf-extensions/pull/85)
- gltf Behavior Graphs Proposal
  - Discord Invite: https://discord.gg/9yjXGTyCjM
- Show and tell
- Create next weeks agenda

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by madjin** [comment]
I was thinking about how our group could participate in plugfests with MSF https://metaverse-standards.org/

The current artist workflow for testing omi khr audio gltf extension (paraphrased)
> spoke scene, using spoke, gltf transform to convert, bring into wordpress app which respects extension, need to update three-omi to factor spec changes, gltf transform spec

@antpb said he'd be down to host a workshop

ultimately need a simple editor to attach OMI audio to something, what might that look like?
Current CLI tools scare artists

If in OMI transform tool can be imported as library
cli would be post process pass from w/e exported unity/blender
small python script post export hook from blender or unity

blender running headless, drag n drop gui on top?
donmccurdy gltf viewer?
https://gestaltor.io/ mentioned, looks interesting

network optimization for downloading (progressive LOD)

checked out humbletims blender / threejs mesh sync program


something about importing positions from source
if there was bonafide gltf delta protocol that'd be cool
refer to gltf + chainset 

third room real time (gltf graph)
plaace that didn't do real time, could support as existing import infra
new revision of scene could be shipped could be full overhaul, keep many different 

giving gltf SDF properties with delta 
gltf isnt for authoring, but if we use as bitstream...
send along diffs, can do time travel things, rewind 

self contained OMI ref, like glbs?

> What is OMI ref in simple terms?
sorta like prefabs
or like js modules system, imports / dynamic imports 

"here's all my runtime optimized assets, how they ref each other, can drop into application, figure out how to correctly fetch them, bundle into different ways"

---

Some notes from last meeting, sorry if they're a bit messy
